packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -1817,3 +1817,125 @@ class RenderSliverToBoxAdapter extends RenderSliverSingleBoxAdapter {
     setChildParentData(child, constraints, geometry);
   }
 }
+
+/// A render object that is invisible during hit testing.
+///
+/// When [ignoring] is true, this render object (and its subtree) is invisible
+/// to hit testing. It still consumes space during layout and paints its sliver
+/// child as usual. It just cannot be the target of located events, because its
+/// render object returns false from [hitTest].
+///
+/// When [ignoringSemantics] is true, the subtree will be invisible to the
+/// semantics layer (and thus e.g. accessibility tools). If [ignoringSemantics]
+/// is null, it uses the value of [ignoring].
+class RenderSliverIgnorePointer extends RenderSliver with RenderObjectWithChildMixin<RenderSliver> {
+  /// Creates a render object that is invisible to hit testing.
+  ///
+  /// The [ignoring] argument must not be null. If [ignoringSemantics], this
```

Comment:
Somehow the second sentence here reads like something there is redundant. :-)

Maybe something like this?: `If [ignore] is true, then [ignoringSemantics] controls whether or not semantics are also ignored.`

Reply:
Oh yeah, this is a little screwy, thanks! It is possible to ignore semantics while _not_ ignoring hit testing in the box equivalent widget. I think these should be documented as independent in both cases then.